docs/static/settings/configuration-management-settings.asciidoc

```diff
@@ -78,3 +78,19 @@ the client’s certificate.
 `xpack.management.elasticsearch.ssl.keystore.password`::
 
 Optional setting that provides the password to the keystore.
+
+`xpack.management.elasticsearch.cloud_id`::
+
+If you're using Elasticsearch cloud, you should specify the identifier here.
+This is an alternative to specifying `xpack.management.elasticsearch.hosts`,
+which should not be used if `cloud_id` is configured.
+This {es} instance will store the Logstash pipeline configurations and metadata.
+
+`xpack.management.elasticsearch.cloud_auth`::
+
+If you're using Elasticsearch cloud, you can set your auth credentials here.
+This is an alternative to having `xpack.management.elasticsearch.username`
+and `xpack.management.elasticsearch.password` - those should not be used if
+`cloud_auth` is configured.
```

Comment:
Suggested wording:

This setting is an alternative to both `xpack.management.elasticsearch.username`
and `xpack.management.elasticsearch.password`. If`cloud_auth` is configured,
those settings should not be used.